sys/include/net/ng_pktbuf.h

```diff
@@ -57,33 +57,37 @@ extern "C" {
 #endif  /* NG_PKTBUF_SIZE */
 
 /**
+ * @brief   Initializes packet buffer module.
```

Comment:
Thats what all the other `_init()` function in RIOT do too in general (except they do something with thread initialization). So should we document this there, too?